qutip/qobj.py

```diff
@@ -953,6 +915,12 @@ def dag(self):
     def dual_chan(self):
         """Dual channel of quantum object representing a completely positive
         map.
+
+        Raises
+        -------
+        ValueError : Dual channels are only implemented for CP maps.
+            If the input is not a completely positive map.
```

Comment:
you don't need to copy in the error message - it just adds in an extra location for us to forget to update it if it changes